### DISCUSSION TITLE
KM271 - no Connection

### DISCUSSION THREAD

**Discussion by florian-ritschel** [dewenni/ESP_Buderus_KM271#127] [General]
Hi, I Updated from v2 to v5, but now I get no connection to my Logamatic.

![grafik](https://github.com/user-attachments/assets/24f4a68a-bb29-4d34-910e-1a253342dcb5)

Any ideas to get it running?

Thanks,
Flo

**Comment by florian-ritschel** [comment]
https://github.com/dewenni/ESP_Buderus_KM271/discussions/86

**Comment by dewenni** [comment]
Have you checked your GPIO Settings?
